shows_app/src/app/(auth)/login/page.tsx

```diff
@@ -1,18 +1,16 @@
-import { Container } from "@chakra-ui/react";
+import { Container, Flex } from "@chakra-ui/react";
 import { LoginForm } from "../../components/feature/auth/LoginForm/LoginForm";
 import { AuthRedirect } from "../../components/feature/auth/AuthRedirect/AuthRedirect";
 
 export default function Login() {
   return (
     <>
-      <AuthRedirect to="/all-shows" condition={"loggedIn"} />
-      <Container
-        backgroundColor="purple.500"
-        padding="10px"
-        justifyContent="center"
-      >
-        <LoginForm />
-      </Container>
+      <AuthRedirect to="/all-shows" condition="loggedIn" />
+      <Flex justifyItems="center">
+        <Container backgroundColor="brand.300">
```

Comment:
This seems pretty similar/the same as the layout used in the /register route, can it be placed in the layout or is there a meaningful difference between the 2?